app/src/main/res/values/strings.xml

```diff
@@ -54,6 +54,7 @@
   <string name="featured_questions">Featured Questions</string>
   <string name="frequently_asked_questions">Frequently Asked Questions</string>
   <string name="faq_activity_title">FAQs (Frequently Asked Questions)</string>
+  <string name="onboarding_activity_title">Onboarding Flow</string>
```

Comment:
Not sure about this--it doesn't seem very user friendly @rt4914. 'Onboarding' isn't really a term we share with users, it's more of a development term. What about 'Oppia introduction' or something similar?

Reply:
`Oppia introduction` seems more correct.